pkg/config/app.go

```diff
@@ -174,6 +177,21 @@ func (c *AppConfig) setEnvVariableConfig() {
 	viper.AutomaticEnv()
 }
 
+// Ensures the version is in the format of v0.0.0
+func formatVersion(version string) string {
```

Comment:
I would avoid formatting here, let's just display what was added during build time as it should be in the correct shape from the start.

Reply:
<img width="469" alt="image" src="https://github.com/user-attachments/assets/0beefd9e-5028-435b-825c-a54439536458">

@mudler A note on the formatting. Right now it's checking for `v0.5.0` but without formatting users (and Masa) will get a variety of variations.  The formatting ensures 3 characters and a trailing 0 if we there isn't another number in that space.

Reply:
I don't think we need this after all.  There was some truncation happening in a few places that was breaking the structure. I adjusted and it seems fine now.